module/mutect2-processes.nf

```diff
@@ -249,7 +220,8 @@ process run_FilterMutectCalls_GATK {
     path "*_filteringStats.tsv"
 
     script:
-    contamination = params.use_contamination_estimation ? contamination_estimation.collect { "--contamination-table '$it'" }.join(' ') : ""
+    tables = contamination_estimation.collect { "--contamination-table '$it'" }.join(' ')
+    contamination = tables == "--contamination-table 'none'" ? "" : tables
```

Comment:
I would recommend changing this check to directly check the input rather than comparing processed strings

Reply:
done